stsci.tools/meta.yaml

```diff
@@ -10,21 +10,23 @@ about:
 
 build:
     number: {{ number }}
+    skip: true  # [py2k]
 
 package:
     name: {{ name }}
     version: {{ version }}
 
 requirements:
     build:
-    - astropy >=1.1
+    - astropy >=2
```

Comment:
astropy is bumped in 2 places - is this intended?

Reply:
Yes. See https://github.com/spacetelescope/stsci.tools/blob/1547f00938d6296028f6bdba242ab8770bdcc5c3/setup.cfg#L30

Reply:
ah thanks